chia/wallet/nft_wallet/nft_puzzles.py

```diff
@@ -58,11 +58,7 @@ def create_nft_layer_puzzle_with_curry_params(
 
 def create_full_puzzle_with_nft_puzzle(singleton_id: bytes32, inner_puzzle: Program) -> Program:
     if log.isEnabledFor(logging.DEBUG):
-        log.debug(
-            "Creating full NFT puzzle with inner puzzle: \n%r\n%r",
-            singleton_id,
-            inner_puzzle.get_tree_hash(),
-        )
+        log.debug("Creating full NFT puzzle with inner puzzle: \n%r\n%r", singleton_id, inner_puzzle.get_tree_hash())
```

Comment:
this `get_tree_hash()` is probably pretty expensive. I believe we call this regardless of whether debug logging is enabled

Reply:
We have it under the condition `if log.isEnabledFor(logging.DEBUG):` so that should be fine.